src/js/components/FileInput/FileInput.js

```diff
@@ -85,6 +86,48 @@ const FileInput = forwardRef(
     const removeRef = useRef();
     const RemoveIcon = theme.fileInput.icons.remove;
 
+    const formatBytes = (size) => {
```

Comment:
this doesn't depend on any outside variable/parameters so it should be moved above the component declaration